### PR TITLE
Task3/aske iza

### PR DESCRIPTION
**Pull request by AskeJL**
- Fixed flake8 errors in pdfminer files
- Moved unittests in ccitt and psparser to seperate files
- Finished task 3

### CODE REVIEW

File: pdfminer/lzw.py [hunk 1]
```diff
@@ -65,12 +65,12 @@ def feed(self, code):
                 x = self.table[code]
             else:
                 raise CorruptDataError
-            l = len(self.table)
-            if l == 511:
+            l1 = len(self.table)
```
Comment (gali8033) [inline pdfminer/lzw.py:68]: I like the renaming

File: pdfminer/lzw.py [hunk 2]
```diff
@@ -96,3 +96,9 @@ def run(self):
 def lzwdecode(data):
     fp = BytesIO(data)
     return b''.join(LZWDecoder(fp).run())
+
+
+if __name__ == '__main__':
```
Comment (gali8033) [inline pdfminer/lzw.py:101]: Should this doctest be used here?

File: pdfminer/utils.py [hunk 3]
```diff
@@ -168,19 +170,19 @@ def choplist(n, seq):
 # nunpack
 def nunpack(s, default=0):
     """Unpacks 1 to 4 byte integers (big endian)."""
-    l = len(s)
-    if not l:
+    length = len(s)
```
Comment (gali8033) [inline pdfminer/utils.py:173]: good renaming

File: tests/test_ccitt.py [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+    def test_m2(self):
+        parser = self.get_parser('101011')
+        parser._do_vertical(-1)
+        parser._do_vertical(-1)
+        parser._do_vertical(1)
+        parser._do_horizontal(1, 1)
+        self.assertEqual(parser._get_bits(), '011101')
+        return
+
+    def test_m3(self):
```
Comment (gali8033) [inline tests/test_ccitt.py:150]: What does b,e,m mean in the naming of the tests?

File: tests/test_pdf2txt.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import unittest
+import tools.pdf2txt
+
+
+class TestPdf2txtMethods(unittest.TestCase):
+
+    def testUsage(self):
+        expected = 100
+        self.assertEqual(tools.pdf2txt.usage(['']), expected)
```
Comment (gali8033) [inline tests/test_pdf2txt.py:9]: Should this line be indented?

File: tests/test_psparser.py [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+        try:
+            while 1:
+                r.append(parser.nextobject())
+        except PSEOF:
+            pass
+        return r
+
+    def test_1(self):
+        tokens = self.get_tokens(self.TESTDATA)
+        print(tokens)
```
Comment (gali8033) [inline tests/test_psparser.py:106]: Maybe we should remove prints :)

File: tests/test_psparser.py [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+        self.assertEqual(tokens, self.TOKENS)
+        return
+
+    def test_2(self):
+        objs = self.get_objects(self.TESTDATA)
+        print(objs)
```
Comment (gali8033) [inline tests/test_psparser.py:112]: same here

**Review by gali8033 — APPROVED** [review]

File: pdfminer/lzw.py [hunk 8]
```diff
@@ -96,3 +96,9 @@ def run(self):
...
+    import doctest
+    print('pdfminer.lzw', doctest.testmod())
```
Comment (ErikJungh) [inline pdfminer/lzw.py:103]: Should this be here?

File: pdfminer/pdffont.py [hunk 9]
```diff
@@ -359,9 +364,9 @@ def __init__(self, name, fp):
             assert 0
         else:
             raise ValueError('unsupported charset format: %r' % format)
-        #print(self.code2gid)
-        #print(self.name2gid)
-        #assert 0
+        # print(self.code2gid)
+        # print(self.name2gid)
+        # assert 0
```
Comment (ErikJungh) [inline pdfminer/pdffont.py:369]: Do we need these comments?

File: pdfminer/pdfinterp.py [hunk 10]
```diff
@@ -268,7 +271,7 @@ def get_inline_data(self, pos, target=b'EI'):
             else:
                 try:
                     j = self.buf.index(target[0], self.charpos)
-                    #print('found', (0, self.buf[j:j+10]))
+                    # print('found', (0, self.buf[j:j+10]))
```
Comment (ErikJungh) [inline pdfminer/pdfinterp.py:274]: Do we need these?

File: pdfminer/pdfinterp.py [hunk 11]
```diff
@@ -727,7 +737,7 @@ def do_Td(self, tx, ty):
         (a, b, c, d, e, f) = self.textstate.matrix
         self.textstate.matrix = (a, b, c, d, tx*a+ty*c+e, tx*b+ty*d+f)
         self.textstate.linematrix = (0, 0)
-        #print('Td(%r,%r): %r' % (tx, ty, self.textstate), file=sys.stderr)
+        # print('Td(%r,%r): %r' % (tx, ty, self.textstate), file=sys.stderr)
```
Comment (ErikJungh) [inline pdfminer/pdfinterp.py:740]: Do we need?

Comment (ErikJungh) [inline pdfminer/pdfinterp.py:740]: Do we need?

File: pdfminer/pdfinterp.py [hunk 12]
```diff
@@ -736,7 +746,7 @@ def do_TD(self, tx, ty):
         self.textstate.matrix = (a, b, c, d, tx*a+ty*c+e, tx*b+ty*d+f)
         self.textstate.leading = ty
         self.textstate.linematrix = (0, 0)
-        #print('TD(%r,%r): %r' % (tx, ty, self.textstate), file=sys.stderr)
+        # print('TD(%r,%r): %r' % (tx, ty, self.textstate), file=sys.stderr)
```
Comment (ErikJungh) [inline pdfminer/pdfinterp.py:749]: Do we need?

File: pdfminer/pdfinterp.py [hunk 13]
```diff
@@ -748,13 +758,14 @@ def do_Tm(self, a, b, c, d, e, f):
...
-        self.textstate.matrix = (a, b, c, d, self.textstate.leading*c+e, self.textstate.leading*d+f)
+        self.textstate.matrix = (a, b, c, d, self.textstate.leading*c+e,
+                                 self.textstate.leading*d+f)
         self.textstate.linematrix = (0, 0)
         return
 
     # show-pos
     def do_TJ(self, seq):
-        #print('TJ(%r): %r' % (seq, self.textstate), file=sys.stderr)
+        # print('TJ(%r): %r' % (seq, self.textstate), file=sys.stderr)
```
Comment (ErikJungh) [inline pdfminer/pdfinterp.py:768]: Do we need?

File: pdfminer/psparser.py [hunk 14]
```diff
@@ -570,12 +576,14 @@ def do_keyword(self, pos, token):
...
-        Arrays and dictionaries are represented as Python lists and dictionaries.
+        Returns keywords, literals, strings, numbers,
+        arrays and dictionaries.
+        Arrays and dictionaries are represented as Python
+        lists and dictionaries.
         """
         while not self.results:
             (pos, token) = self.nexttoken()
-            #print((pos,token), (self.curtype, self.curstack))
+            # print((pos,token), (self.curtype, self.curstack))
```
Comment (ErikJungh) [inline pdfminer/psparser.py:586]: : -- :

**Review by ErikJungh — APPROVED** [review]
Some commented prints that should probably be removed

File: tests/test_pdf2txt.py (hunk 5 above)
Comment (AskeJL) [reply tests/test_pdf2txt.py:9]: Yes acording to flake8

File: tests/test_psparser.py (hunk 7 above)
Comment (AskeJL) [reply tests/test_psparser.py:112]: Same as above.

**Pull request merged**